src/routing/queryParams.ts

```diff
@@ -1,6 +1,9 @@
 // TODO: rename QueryParams https://en.wikipedia.org/wiki/Query_string
 enum QueryParams {
     routes= 'routes',
+    routeId= 'routeid',
+    startTime= 'starttime',
+    direction= 'direction',
```

Comment:
Why are startTime and direction params added instead of routePathId?

I know that routePathId is generated at our frontend.

I see 2 ways of fixing this:

Option 1: Make routePathId format so that our frontend can decode it. So don't use hash when gererating it.

Option 2: What if our backend generated routePathId and we could query a specific routePath from our backend with that id?

Reply:
I don't see this as a problem. The primary keys of routePaths in Jore are routeId, startTime and direction. When fetching data from the backend, we need to use these primary keys. As you told, we cannot use any hash, but I don't see the need for encoding these primary keys, and later decoding them.
I don't see any problem with setting all these values in the URL, and I think that is a better solution than having some made up ID, that we need to encode/decode.

Reply:
I see. The reason I said about this that now we refer to a routePath with 2 ways, our generated routePathId and routeId+startTime+direction. I wish there was only one way but maybe it would also make our lives more difficult. So maybe i'm fine with this solution too.